### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by vikitaotiz**
# We have implemented the following features and activities;

1. Config & basic setup
2. Redux: Fetch data and update Redux store
3. Render UI: lists
4. Redux: Write actions and reducers for booking rockets/dragons and joining missions
5. Redux: Write actions and reducers for canceling rockets/dragons and leaving missions
6. Render UI: conditional components rendering
7. Render UI: My Profile section

### CODE REVIEW

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi Team 🙋‍♀️, 

Great job so far! 👏 

Pull request is 👌 
Git flow ✔️ 
README is 🎯 

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- LinkedIn: [LinkedIn](https://linkedin.com/in/henry-kc)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to contribute
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
[the remark this answers is not in the document]
Comment (HENRYKC24) [reply README.md:54]: Sure

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        </NavLink>
+      </li>
+      <li className="pipe">|</li>
+      <li>
+        <NavLink to="/profile" exact>
+          My Profile
+        </NavLink>
+      </li>
+    </ul>
+  </nav>
```
[the remark this answers is not in the document]
Comment (HENRYKC24) [reply src/components/Navbar.js:30]: Consider it done, @AshakaE:ok_hand:
Thank you very much.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @vikitaotiz and @HENRYKC24, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**